src/niscope/system_tests/test_system_niscope.py

```diff
@@ -1,527 +1,588 @@
 import collections
 import fasteners
+import grpc
 import hightime
 import math
 import niscope
 import numpy
 import os
+import pathlib
 import pytest
+import subprocess
 import tempfile
-
```

Comment:
To get a decent diff for this file, I'd recommend hiding whitespace changes:

![image](https://user-images.githubusercontent.com/77176215/199837556-0bce6876-e347-4120-9ff9-5aba4b10a941.png)